sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -151,6 +175,73 @@
             });
     }
 
+    private Flux<Instant> renewMessageLock(ServiceBusReceivedMessage[] messagesforRenewLock) {
+
+        return  cbsAuthorizationOnce()
+            .then(
+                channelMono.flatMap(requestResponseChannel -> {
+                    UUID[] lockTokens = new UUID[messagesforRenewLock.length];
+                    int index = 0;
+                    for (ServiceBusReceivedMessage receivedMessage : messagesforRenewLock
+                    ) {
+                        lockTokens[index++] = receivedMessage.getLockToken();
+                    }
+                    HashMap<String, UUID[]> requestBodyMap = new HashMap<>();
+                    requestBodyMap.put(REQUEST_RESPONSE_LOCKTOKENS, lockTokens);
+                    Message requestMessage = createRequestMessageFromValueBody(REQUEST_RESPONSE_RENEWLOCK_OPERATION,
+                        requestBodyMap, MessageUtils.adjustServerTimeout(Duration.ofSeconds(60)));
```

Comment:
Given this is a singleton map could this use `Collections.singletonMap`?

```suggestion
Message requestMessage = createRequestMessageFromValueBody(REQUEST_RESPONSE_RENEWLOCK_OPERATION,
    Collections.singletonMap(REQUEST_RESPONSE_LOCKTOKENS, lockTokens),
    MessageUtils.adjustServerTimeout(Duration.ofSeconds(60)));
```